packages/terra-form-select/src/Combobox.jsx

```diff
@@ -88,10 +88,6 @@ const propTypes = {
    * Callback function invoked for each option on search change. function(searchValue, option)
    */
   optionFilter: PropTypes.func,
-  /**
-   * Placeholder text.
-   */
-  placeholder: PropTypes.string,
```

Comment:
We could consider leaving this and logging a depreciation notice when used. Otherwise people will start getting unused prop warnings.

Reply:
Lets go ahead and make this change.

Reply:
Does [this](https://github.com/cerner/terra-core/pull/3216/files#diff-04913f4c92d4b79cc19b6a7dac3a597d7b4af579264bdc4ed0c16214bf8c32c0R173) look good?